src/controllers/post.js

```diff
@@ -1,28 +1,28 @@
 import { sendDataResponse } from '../utils/responses.js'
+import Post from '../domain/post.js'
 
 export const create = async (req, res) => {
   const { content } = req.body
-
   if (!content) {
     return sendDataResponse(res, 400, { content: 'Must provide content' })
   }
+  try {
+    const postToCreate = await Post.fromJson([req.body, req.user.id])
 
-  return sendDataResponse(res, 201, { post: { id: 1, content } })
+    const posts = await postToCreate.save()
+    return sendDataResponse(res, 201, { ...posts })
```

Comment:
You don't need to use the spread operator here, you can just do: 

```js
return sendDataResponse(res, 201, posts)
```

Although I would suggest renaming `posts` to `post` as we're just creating one, not multiple